### PR TITLE
JavaScript capstone project - API-based webapp

### PR DESCRIPTION
**Pull request by ShoiraTa**
## Interfaces:
- A home page, that shows a list of items that you can "like."
- A popup window with more data about an item where user can leave comments

## API : MealDB
- Application gets a list of items with a unique item id
- Using id of the given item, application posts detailed information about the item on the comments window.

## Counters: 

- The number of items (home).
- The number of comments (comments popup).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import './style.css';
```
Comment (SajjadAhmad14) [inline src/index.js:1]: Please follow the wireframe for the navigation menu. The logo should be on the left side of the navigation items.

Home page footer text should be on the left side of as shown in the mock-up.

File: src/meal-api.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
+import { like, Displaylikes } from './interactions.js';
```
Comment (SajjadAhmad14) [inline src/meal-api.js:1]: Please record a new video that only highlights a piece of the code that you find interesting or are proud of making. You do not need to present the whole project again.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Danny and Shoira, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Nice and professional README. :heavy_check_mark:
Git Flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 3]
```diff
@@ -27,17 +48,21 @@ https://timowest12.github.io/Awesome-books/
...
+![Daniel's GitHub stats](https://github-readme-stats.vercel.app/api?username=danieldotcomcoder&count_private=true&theme=dark&show_icons=true)
 
 ## Acknowledgments
 
 - Microverse
 
 ## 📝 License
 
-This project is [MIT](./MIT) licensed.
+This project is [MIT](<.(https://choosealicense.com/licenses/mit/)>) licensed.
```
Comment (mikethreels) [inline README.md:64]: - [ ] [OPTIONAL] If you want to make your project look even more professional you might want to consider adding the MIT.md file to your repo and adding a link to it in your Readme.md file like is done in [the template provided by Microverse](https://github.com/microverseinc/readme-template).

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Great job on the requested suggestions by the previous CR! 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

- [OPTIONAL] In case you're interested I've added an optional inline comment which might make your project look even more professional than it already does 💯 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**